### PR TITLE
Mayhem Integretion PR

### PR DESCRIPTION
**Pull request by rishabh1024**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:18.04 as base
```
Comment (xansec) [inline Dockerfile:1]: Can you please separate this Dockerfile into multiple stages (build and package)? You also do not necessarily need to set the CMD or ENTRYPOINT macros.

File: Dockerfile.bak [hunk 2]
```diff
@@ -7,8 +7,18 @@ RUN apk add --update alpine-sdk openssl-dev
```
Comment (xansec) [inline Dockerfile.bak]: This is the original Dockerfile. Please do not modify upstream Dockerfiles. I recommend renaming your fuzzing Dockerfile to something like `Dockerfile.mayhem`, and referencing the Dockerfile in your mayhem.yml, and reverting the changes to the original Dockerfile.

File: Makefile [hunk 3]
```diff
@@ -119,10 +119,9 @@ test: test_build
```
Comment (xansec) [inline Makefile:119]: Can you explain why these changes were necessary? It is best to make modifications to support fuzzing as optional (for example by protecting it with a macro or IFDEF).

File: Mayhemfile [hunk 4]
```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/rishabh1024/ffproxy-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit the image name here, as the image name gets populated during the workflow run.

**Review by xansec — CHANGES_REQUESTED** [review]
Please do not remove files unless necessary. I recommend restoring the corpora that you removed. Please make the requested changes.

File: Makefile (hunk 3 above)
Comment (rishabh1024) [reply Makefile:119]: I did not make any change here. It was in the original file as well.

File: Dockerfile.bak (hunk 2 above)
Comment (rishabh1024) [reply Dockerfile.bak]: I have removed the Dockerfile.bak and restored it back to prod.Dockerfile in the docker folder. Now my own file is named Dockerfile.
.

File: Mayhemfile (hunk 4 above)
Comment (rishabh1024) [reply Mayhemfile]: Completed.

File: Makefile (hunk 3 above)
Comment (xansec) [reply Makefile:119]: Ok, it looks like just a path difference anyways.
